test/utils/Precompiles.t.sol

```diff
@@ -25,7 +25,7 @@ contract Precompiles_Test is Base_Test {
     function test_DeployLockupDynamic() external onlyTestOptimizedProfile {
         address actualLockupDynamic = address(precompiles.deployLockupDynamic(users.admin, nftDescriptor));
         address expectedLockupDynamic =
-            address(deployOptimizedLockupDynamic(users.admin, nftDescriptor, defaults.MAX_COUNT()));
+            address(deployOptimizedLockupDynamic(users.admin, nftDescriptor, defaults.MAX_SEGMENT_COUNT()));
```

Comment:
Precompile tests are failing because this is not same as the values we are using in `Precompiles.sol`.

Reply:
I think we should keep it 500 until https://github.com/sablier-labs/v2-core/issues/872 is resolved.

Reply:
Good catch and OK, I will revert to 500.